parsec/core/types/manifest.py

```diff
@@ -670,6 +676,9 @@ def to_stats(self):
     def parent(self):
         return self.base.parent
 
+    def size_estimate(self):
+        return 1 + len(self.base.children) + len(self.children)
```

Comment:
I don't really get the logic here. Why ` len(self.base.children) + len(self.children)` is not enough

Also, it seems that this logic (introduced in [this commit](https://github.com/Scille/parsec-cloud/commit/7d54659e759276b37a85dac80ddb8c1f5d1d45f6) ) is not used anymore (removed [there](https://github.com/Scille/parsec-cloud/commit/f8294eccb121a12bb523302ccda4dc8fff9205e9#diff-0a806323e83144ea32278ce2ffd433c2af0473ac5095bd58d2788b336b8931ec) )

Reply:
Good catch! I left it on purpose so we can discuss its usefulness, as this estimate can be useful for other serde operation in the code (for remote manifests for instance).

The size estimate is an estimate of the number of entries that the manifest is referencing (as it should grow roughly linearly with the serialization time). I'm considering the manifest itself as an entry so empty file/folder/workspace manifests have the same size as a user manifest, i.e. 1. 

It's quite arbitrary though, and it can be changed/removed.

Reply:
0ac8ce05b053026fdfec8e7242a425a82b0f47bc